tests/rptest/tests/datalake/databricks_test.py

```diff
@@ -230,6 +231,97 @@ class TestCase:
                 assert actual_schema == tc.dbx_schema, \
                     f"Expected DBX schema {tc.dbx_schema} but got {actual_schema}"
 
+    @cluster(num_nodes=2)
+    @matrix(cloud_storage_type=supported_storage_types())
+    def test_e2e_with_partition_evolution(self, cloud_storage_type):
+        # TODO: Move this in the matrix decorator. Somehow.
+        if not DatabricksContext.available(self.test_context):
+            self.logger.warning(
+                "Skipping test because Databricks context is not available")
+            cleanup_on_early_exit(self)
+            return
+
+        if get_cloud_provider() != "aws":
+            self.logger.warning(
+                f"Skipping test because it is only supported on AWS, but the current cloud provider is {get_cloud_provider()}"
+            )
+            cleanup_on_early_exit(self)
+            return
```

Comment:
nit: probably makes sense to wrap this into some `def supports_dbx(test_context) -> tuple[bool, str]` (bool: is_supported, str: warning_msg) or somesuch? I imagine this condition will be the same across all dbx tests

Reply:
Yes